grpc.sh

```diff
@@ -40,7 +40,7 @@ env -i HOME="$HOME"                             \
 
 # ldconfig won't work if we're not running make install as root, and that's ok,
 # we don't need it
-sed -i 's/ldconfig/true/' ./Makefile
+sed -i -e 's/ldconfig/true/' Makefile
```

Comment:
Unfortunately this won't work on macOS, where you need to provide a zero-length extension to `-i`. The way you wrote it creates `Makefile-e` as backup, since `-e` is seen as the extension.

One universal (BSD/GNU) way of fixing the issue is:

```bash
sed -i.deleteme -e 's/ldconfig/true/' Makefile
rm -f *.deleteme
```

Can you fix it please? Thanks!

Reply:
Sure, fixing immediately, thank you for the pointer :)